.github/scripts/beta-docker-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+package_json_version=$(grep '"version":' package.json | cut -d ':' -f 2- | tr -d ' ' | tr -d '"' | tr -d ',')
+beta_feature=$(echo $package_json_version | sed -r 's/[0-9]+.[0-9]+.[0-9]+-//')
+beta_feature=$(echo $beta_feature | sed -r 's/-beta\.[0-9]*$//')
```

Comment:
Why do it in 2 lines? It's a bit confusing, isn't it?

Reply:
I think it helps in readability